packages/examples/src/assets/openstreet-map-style.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 8,
+    "name": "Basic",
+    "sources": {
+        "osm-tiles": {
+            "type": "raster",
+            "tiles": [
+                "https://a.tile.openstreetmap.org/{z}/{x}/{y}.png",
```

Comment:
Is this completely open? No api key required? Is a vector tile source available?

Reply:
Yes I tested it, it works - no need for api key.
I don't know what a vector tile source is though, I'll be honest, I asked AI to create this so I'm not sure how it works exactly.